recipe/meta.yaml

```diff
@@ -10,19 +10,19 @@ source:
   sha256: b403d3dad4ef542fdc3024fa37d3a6cedb4ad33c72e31b6d9bab89dcaf69edf7
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ stdlib("c") }}
     - cmake
-    - make  # [unix]
+    - make                # [not win]
+    - ninja               # [win]
```

Comment:
We _could_ just use ninja on both platforms I think?

What do you think @versatran01 ?

Reply:
Only ninja is not the "usual" way on conda-forge following knowledge base doc.
The combo make not win ninja is classic on conda-forge cpp-recipes